gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=7197a12f450794931532469d4ff21a59ea2c1cd59a3ec3f89c035c3c420a6999
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.14.2-bin.zip
```

Comment:
I've bumped Gradle in order to be able to edit the project using Java 25. Please let me know if this needs some additional consideration, or I can just revert this part